### ISSUE TITLE
PD011 false positives for other objects and method calls

### ISSUE THREAD

**Issue by guyrosin** [charliermarsh/ruff#1468]
PD011 (UseOfDotValues) triggers for `.values` attribute access and `.values()` method calls for objects that are not dataframes. Examples:

```py
{"hello": 1}.values
{"hello": 1}.values()
```

**Comment by edgarrmondragon** [comment]
Unfortunately this an shared issue with `pandas-vet`: https://github.com/deppen8/pandas-vet/issues/74#issuecomment-582329625.

I think the best we can do is not trigger on the second case (i.e. method call).

**Comment by charliermarsh** [comment]
I added some heuristics that should help with this case in #1538. But it will still be a problem in some cases.
